app/Controllers/Http/DeletesController.ts

```diff
@@ -0,0 +1,27 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from 'App/Models/Profile'
+
+export default class DeletesController {
+    public async deleteUser({ request, auth, response }: HttpContextContract) {
```

Comment:
Controller should be differentiated based on Table name. For eg: ProfileController.ts, UsersController.ts

Reply:
Done